android/cli/commands/_build.js

```diff
@@ -1657,6 +1667,10 @@ process.exit(1);
 		 		}
 		 		logger.error(msg);
 		 		process.exit(1);
+		 		*/
+
+		 		// re-validate modules
```

Comment:
Is there a possible infinite loop here? I see before you'd just give up and report missing dependencies, but now we basically implicitly add them for you and re-try.

Reply:
- It looks like that, but our existing checks will prevent that from happening and return before reaching this code again